packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -53,9 +53,9 @@ export interface IOdspSocketError {
 
     /**
      * Optional Retry-After time in seconds
-     * The client should wait this many seconds before retrying its request
+     * The client should wait this many milliseconds before retrying its request
      */
-    retryAfter?: number;
+    retryAfterMs?: number;
```

Comment:
is this a contract? specifically do we use this type to deserialize responses from the server? or is this something we build in the driver?